packages/core/src/Site/siteConvertLayout.njk

```diff
@@ -38,13 +38,12 @@
   </nav>
 </div>
 
-{% set defaultFooter %}
```

Comment:
Is there any issue with making the `{%-` change here? I see that you moved this into `<footer>`

Reply:
As shown in the summary of changes I made, `{%-` is intentional and is to remove the newlines before the `{% set %}` tag.